### PR TITLE
fix alignment issues

### PR DESCRIPTION
**Pull request by kiwansim**
center these two buttons
Screenshot 2024-03-28 at 2 01 18 PM

left align get repairs 
Screenshot 2024-03-28 at 2 00 19 PM

update fonts to match mock, fix alignment with "new link sent" and svg
Screenshot 2024-03-28 at 2 01 01 PM
Screenshot 2024-03-28 at 2 00 49 PM

### CODE REVIEW

**Comment by kiwansim** [comment]
@lesliejustfix for your review! deploy preview is [here](https://deploy-preview-882--wow-django-dev.netlify.app/en/)

File: client/src/styles/AccountSettingsPage.scss [hunk 1]
```diff
@@ -73,8 +73,10 @@
       display: flex;
       flex-direction: column;
       gap: 0.625;
+
       .send-new-link {
         justify-content: start;
+        align-items: center;
```
Comment (austensen) [inline client/src/styles/AccountSettingsPage.scss:79]: I'm not seeing that this does anything - wasn't it only the `link-sent` change that was needed?

**Review by austensen — APPROVED** [review]
Looks great!

File: client/src/styles/AccountSettingsPage.scss (hunk 1 above)
Comment (kiwansim) [reply client/src/styles/AccountSettingsPage.scss:79]: yeah good catch!

**Pull request merged**